packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -409,13 +409,13 @@ async function load_shadow_data(route, event, options, prerender) {
 	try {
 		const mod = await route.shadow();
 
```

Comment:
```suggestion

	// TODO: Remove for 1.0
	['get', 'post', 'put', 'patch', 'del'].forEach((m) => {
		if (mod[m] !== undefined) {
			const replacement = m === 'del' ? 'DELETE' : m.toUpperCase();
			throw Error(`Endpoint method "${m}" has changed to "${replacement}"`);
		}
	});

```

The error should be here as well?

Reply:
ah yep, i'll rustle up a utility that can be used in both places